client/src/components/searchResults/rows.js

```diff
@@ -23,9 +24,33 @@ const tableRow = profile => {
         </div>
         <div className="search-table-icon search-table-border">
           <div className="search-table-avatar">{avatar}</div>
-          <FlatButton className="search-table-flatbutton" hoverColor="#e7e0d7">
-            View profile
-          </FlatButton>
+          {type === 'User' &&
+            <FlatButton 
+              className="search-table-flatbutton" 
+              hoverColor="#e7e0d7"
+              containerElement={<Link to={`/roadmap/${id}`} />} 
```

Comment:
has this route been added already?

Reply:
Yeah, the /roadmap/id route was added with the display_roadmaps_80 branch was merged.